.circleci/config.yml

```diff
@@ -35,3 +42,13 @@ workflows:
         filters:
           branches:
             only: master
+  nightly:
+    triggers:
+      - schedule:
+          cron: "0 0 * * *"
```

Comment:
Let's randomize this value a bit so we avoid running everything at midnight. :smile:

Reply:
Bumped to 6am as Go releases happen at PST times usually.